tac/project-updates/2025/2025-annual-Identus.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: 2025 Annual Review Identus
+parent: 2025
+grand_parent: Project Updates
+---
+
+## Project Health
+
+LF Decentralized Trust Identus [LFX Insights Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/reports/contributors?project=identus&bestPractice=false&dateFilters=Last%20Year&dateRange=2024-01-01%20to%202024-12-31&compare=PP&granularity=month&hideBots=true&repository=all)
+The project is relatively new to the LF Decentralized Trust Ecosystem and is mainly supported by the IOG maintainers.
+At the same time, over the past year, we have seen several new contributors and evidence of usage of the Identus platform by other companies: 
+- https://socious.io/, Socious production use case.
+- https://ahau.io/, Ahau PoC.
+- https://blocktrust.dev/, - identity wallet.
```

Comment:
This looks great. Please consider adding these references to an `ADOPTERS.md` file in your main technical repo (preferably not the `governance` one, which is for administration). Soon, we will have an official recommendation for the presence of this file in the TAC's guidelines (there's an active [PR](https://github.com/LF-Decentralized-Trust/governance/pull/79) on that.)